src/Ztm.Data.Entity.Postgres/MainDatabase.cs

```diff
@@ -25,19 +26,16 @@ public MainDatabase(DbContextOptions<Ztm.Data.Entity.Contexts.MainDatabase> opti
         {
         }
 
-        protected override void ConfigureWebApiCallback(ModelBuilder modelBuilder)
+        protected override void ConfigureWebApiCallback(EntityTypeBuilder<WebApiCallback> builder)
         {
-            base.ConfigureWebApiCallback(modelBuilder);
-
-            modelBuilder.Entity<WebApiCallback>(b =>
-            {
-                b.Property(e => e.Url).HasConversion(Converters.UriToStringConverter);
-            });
+            base.ConfigureWebApiCallback(builder);
```

Comment:
We should have empty line between between base method invocation so it more easy to read.